modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js

```diff
@@ -14,6 +14,64 @@
 
 Liferay = window.Liferay || {};
 
+Liferay.Webpack = {
+	Container: {},
+
+	SharedScope: {},
+
+	require: function(moduleName, callback) {
+		var parts = moduleName.split("/");
```

Comment:
This isn't getting backported, right? And it is transpiled anyway, so you should feel free to use `const`/`var` here instead of `var` (you're already using arrow functions anyway). I know there are `var` elsewhere in the file, but we may as well gradually edge towards newer syntax when we add new code.

Reply:
:+1: